Tests/ApolloCodegenTests/CodeGeneration/Templates/SelectionSet/SelectionSetTemplateTests.swift

```diff
@@ -4794,7 +4794,7 @@ class SelectionSetTemplateTests: XCTestCase {
       AllAnimals: [Animal!]
     }
 
-    type Animal {
+    type Animal @typePolicy(keyFields: "id") {
```

Comment:
Why did we need to add this here if we aren't altering any of the tests or adding any new ones?

Reply:
This was from an earlier revision, this change isn't supposed to be here (yet)